### ISSUE TITLE
Negative LibYear Totals are returned

### ISSUE THREAD

**Issue by nickiemc** [corgibytes/freshli#78]
In some instances, a negative LibYear value is returned. LibYear should never be negative.

For example, when running against [https://github.com/thoughtbot/clearance](https://github.com/thoughtbot/clearance), the follow values are returned: 

```
2011/11/01	-22.151
2011/12/01	-16.214
2012/01/01	-15.008
2012/02/01	-9.893
2012/03/01	-5.575
2012/04/01	0.132
...
```

**Comment by nickiemc** [comment]
After some initial investigation, I shared my initial findings in Slack: https://corgibytes.slack.com/archives/C04AUUJ2N/p1596665824154400

**Comment by nickiemc** [comment]
Initial issues were fixed in PR #85. Additional negative values exist, but issues #105 and #113 have been created to address these. Additional issues can be created as needed.
